spec/bolt/outputter/human_spec.rb

```diff
@@ -285,4 +285,15 @@
     outputter.fatal_error(Bolt::CLIError.new("oops"))
     expect(output.string).to eq("oops\n")
   end
+
+  it "handles message events" do
+    outputter.print_message_event(message: "hello world")
+    expect(output.string).to eq("hello world\n")
+  end
+
+  it "colorizes message events" do
```

Comment:
I think removing the color is a good idea. Need to update the test.

Reply:
Ugh I missed that test.